### PR TITLE
T12082 resource rename 

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: refactoring
* Link to issue: #12082 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Renamed `Phalcon\Assets\Resource` and related methods to `Assets`
Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -44,6 +44,13 @@
 - Changed `Phalon\Tag::textArea` to use `htmlspecialchars` to prevent XSS injection. [#12428](https://github.com/phalcon/cphalcon/issues/12428)
 - Changed `Phalon\Cache\Backend\*::get` to use only positive numbers for `lifetime`
 - Changed `Phalcon\Logger` to comply with PSR-3. The component has been rewritten to use adapters that alllow logging to different areas. The [#13438](https://github.com/phalcon/cphalcon/issues/13438)
+- Renamed `Phalcon\Assets\Resource` namespace to `Phalcon\Assets\Asset` [#12082](https://github.com/phalcon/cphalcon/issues/12082)
+- Renamed `Phalcon\Assets\Resource` to `Phalcon\Assets\Asset` [#12082](https://github.com/phalcon/cphalcon/issues/12082)
```
Comment (sergeyklay) [inline CHANGELOG-4.0.md]: Duplicated change log entries  
- Renamed `Phalcon\Assets\Resource` namespace to
- Renamed `Phalcon\Assets\Resource` to

**Review by sergeyklay — DISMISSED** [review]
In general LGTM.

File: CHANGELOG-4.0.md (hunk 1 above)
Comment (niden) [reply CHANGELOG-4.0.md]: Thanks I will fix the changelog later.

**Pull request merged**